harness/determined/pytorch/deepspeed/dsat/_defaults.py

```diff
@@ -43,36 +44,11 @@
     },
 }
 
-AUTOTUNING_DICT = {
-    "enabled": True,
-    "results_dir": "autotuning_results",
-    "exps_dir": "autotuning_exps",
-    "overwrite": False,
-    "metric": "throughput",  # TODO: dynamically populate based on searcher fields.
-    "start_profile_step": 3,
-    "end_profile_step": 5,
-    "fast": True,
-    "max_train_batch_size": None,
-    "mp_size": 1,
-    "num_tuning_micro_batch_sizes": 5,
-    "tuner_type": "model_based",
-    "tuner_early_stopping": 25,
-    "tuner_num_trials": 25,
-    "arg_mappings": None,
-}
+AUTOTUNING_DICT = {"autotuning": {"enabled": True}}
```

Comment:
What happened to a bunch of these parameters? Perhaps I'm mistaken but from what I understand, we were essentially trying adapt these values but make them available on the command line? 

A bunch of them seem to be missing, like `fast` and `mp_size` among others. Is this intended and how are we making the judgement call of what is here?

Reply:
Yeah, so all of these fields are the ones used by DS to configure their own autotuner, but some of them don't correspond to any functionality we actually provide (we were just ignoring them), while others represents concepts we already have established names for.  So, I cut out everything but the bare minimum you need to get DS autotuning to function.

Didn't want people seeing like `fast` set to some value and thinking it was meaningful, because it wasn't.  Just copied in all the defaults a while ago.